src/login_nopam.c

```diff
@@ -263,18 +265,40 @@ static bool user_match (const char *tok, const char *string)
 
 static const char *resolve_hostname (const char *string)
 {
-	/*
-	 * Resolve hostname to numeric IP address, as suggested
-	 * by Dave Hagewood <admin@arrowweb.com>.  --marekm
-	 */
-	struct hostent *hp;
+	int                  gai;
```

Comment:
I think it would make more sense to rename this variable. Something like `gai_error` would make more sense.

Reply:
Yep.  I used gai_err, to not make a line too long.